velox/exec/tests/utils/OperatorTestBase.cpp

```diff
@@ -113,11 +113,22 @@ void OperatorTestBase::SetUp() {
   }
   driverExecutor_ = std::make_unique<folly::CPUThreadPoolExecutor>(3);
   ioExecutor_ = std::make_unique<folly::IOThreadPoolExecutor>(3);
+  PeriodicStatsReporter::Options options;
+  options.allocator = memory::memoryManager()->allocator();
+  options.allocatorStatsIntervalMs = 2'000;
```

Comment:
Most test run quickly and how about we set it to a lower frequency? Do we log a lot in stats reporter? If it is the case, let keep as 2s to avoid too much log message in test log. Thanks!

Reply:
Logging is as following:
1 line upon start
1 line upon stop
1 line for spill pool

I'd say most of the tests probably only will have stats reporter report only 1 time (upon start). The second interval likely won't kick in unless we set it super frequent. There is a big downside of doing so: log spamming from spill pool for some long running tests. If the test runs long, then 2 seconds should be sufficient to test the reporter to report multiple times.